SecurityManager/Metadata.Importer/Program.cs

```diff
@@ -93,15 +94,15 @@ public int Run ()
         ITypeDiscoveryService typeDiscoveryService = new AssemblyFinderTypeDiscoveryService(assemblyFinder);
         MappingConfiguration.SetCurrent(
             new MappingConfiguration(
-                new MappingReflector(
+                MappingReflector.Create(
                     typeDiscoveryService,
                     new ClassIDProvider(),
                     new ReflectionBasedMemberInformationNameResolver(),
                     new PropertyMetadataReflector(),
                     new DomainModelConstraintProvider(),
                     new PropertyDefaultValueProvider(),
                     new SortExpressionDefinitionProvider(),
-                    MappingReflector.CreateDomainObjectCreator()),
+                    SafeServiceLocator.Current.GetInstance<IDomainObjectCreator>()),
```

Comment:
Use IoC to get the parameter values, please. Once we got ITypeDiscoverySerivce also set up via IoC, we probably will change the two security assemblies to use a custom registration for the AssemblyFinder and just get the MappingReflector completly via IoC.